apps/fluent-tester/src/TestComponents/Badge/BasicBadgeTest.tsx

```diff
@@ -68,11 +60,27 @@ export const BasicBadge: React.FunctionComponent = () => {
   const badgeConfig = {
     appearance: badgeAppearance,
     badgeColor,
-    position: 'absolute' as FlexStyle['position'],
     size,
     shape,
+    shadowToken: showShadow ? theme.shadows.shadow4 : undefined,
   };
 
+  const StyledBadge = memoize(
```

Comment:
I think this will run the code to memoize the customize function on every render, so I don't think this will cache the results between renders like you're intending it to.

Reply:
I added useMemo in function